1.8/administration/installing/haproxy-adminrouter.md

```diff
@@ -1,10 +1,10 @@
 ---
-post_title: Configuring HAProxy in Front of Admin Router
-nav_title: HAProxy in Front of Admin Router
-menu_order: 3.3
+post_title: Configuring a Custom Certificate
+menu_order: 6
 ---
 
-You can use HAProxy to set up an HTTP proxy in front of the DC/OS [Admin Router](/docs/1.8/overview/concepts/#adminrouter).
+You can use HAProxy to set up an HTTP proxy in front of the DC/OS [Admin Router](/docs/1.8/overview/concepts/#adminrouter). This can be useful if you want to put SSL with your own certificate on top of DC/OS. DC/OS does not support adding your own certificates directly into Admin Router.
```

Comment:
`DC/OS does not currently support adding your own certificates directly into Admin Router.`